### DISCUSSION TITLE
Security?

### DISCUSSION THREAD

**Discussion by ValkorionBaratheon** [jhassan8/crunchyroll-tizen#69] [Q&A] [answered]
Hi, I was just wondering how security of our login information is handled? 

I would really love to use this for my Samsung TV, but just want to be aware of any potential security risks. Thanks!

**Comment by jhassan8** [comment, marked as answer]
Hi @ValkorionBaratheon, currently the app only has the login option by user and password. Later I will try to add the option to activate the device.

But as for your question, the user data is not accessible to anyone, it is only on the TV and the app only uses official crunchyroll APIs.

**Reply by ValkorionBaratheon** [reply]
Thank you so much!

**Reply by BlackwolfSGP** [reply]
@jhassan8 
Hi, adding to this question, I would like to ask if traffic is encripted in the api calls that make login possible? Is it simple https or is there a layer of encryption the api allows?

**Reply by jhassan8** [reply]
Hi @BlackwolfSGP, all the security that the crunchyrroll APIs have, the app only consumes official crunchyroll APIs, and does not store data, nor access anything external. I hope I have answered your questions...
